lib/skunk/cli/commands/status_reporter.rb

```diff
@@ -66,9 +66,11 @@ def stink_score_average
       end
 
       def table_options
+        max = sorted_modules.max_by { |a_mod| a_mod.pathname.to_s.length }
+        width = max.pathname.to_s.length + 77
```

Comment:
What is the `77` magic number?

Reply:
@bronzdoc This PR makes the width variable. Width is determined by "longest file path" + 74 characters.

```
$ irb
2.6.3 :001 > "| stink_score  | churn_times_cost | churn      | cost       | coverage   |".size
 => 74
```

I guess I could change it so `77` is relative to the right side of the header?

Reply:
Maybe we could put that 77 in a constant? `EXTRA_HEADER_LENGTH` ?
```ruby
width = max.pathname.to_s.length + EXTRA_HEADER_LENGTH
```

Reply:
@bronzdoc Good idea. I'll do that later today.